lib/exe/lib/Cardano/Wallet/Application/Logging.hs

```diff
@@ -27,13 +30,19 @@ import GHC.Generics
 data ApplicationLog
     = ApiApplicationLog ApiApplicationLog
     | MsgServerStartupError ListenError
+    | UiApplicationLog String
```

Comment:
```suggestion
    | UIApplicationLog String
```

I never know how to capitalize acronyms in Haskell constructors (`HTTP` vs `Http` 🙈 ). My gut feeling tells me that `UI` instead of `Ui` may be the better choice here.